### DISCUSSION TITLE
remote ruby podcast, pagy docs

### DISCUSSION THREAD

**Discussion by benkoshy** [retypeapp/retype#514] [Show and tell]
First listen here: https://remoteruby.com/222#t=0h13m39s

Did you listen? You didn't? Do not read any further till you listen for 3-4 minutes on that time stamp.

Check out the docs here: https://ddnexus.github.io/pagy/

Point to note: 

- IMO: Open source documentation is the gateway drug towards commercial licensing.

**Comment by geoffreymcgill** [comment]
Nice. Thanks for sharing!

**Comment by TressieApplenwj** [comment]
Thanks for sharing!

**Comment by benkoshy** [comment]
here again: https://github.com/ddnexus/pagy/discussions/534
